src/main/java/com/crafting/KnowledgeCreationException.java

```diff
@@ -0,0 +1,14 @@
+package com.crafting;
+
+/**
+ * Created by marvin on 02.11.16.
+ */
+public class KnowledgeCreationException extends Exception
+{
+    static final long serialVersionUID = -3387516993124229948L;
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Explicitly declare the visibility for "serialVersionUID". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS2039)